rebasehelper/application.py

```diff
@@ -351,7 +351,7 @@ def patch_sources(self, sources):
         git_helper = GitHelper(sources[0])
         if not self.conf.non_interactive:
             git_helper.check_git_config()
-        patch = Patcher(self.conf.patchtool)
+        patch = Patcher("git")
```

Comment:
Ahh, I was not able to find anything like this, thank you for comment, I will fix it ASAP.